ui/app/common/pnc-client/pagination/filteringPaginator.js

```diff
@@ -204,9 +211,28 @@
          */
         that.clearSort = function () {
           sortBy = null;
+
+          handleSortChangeCallbacks();
           return this;
         };
 
+        /**
+         * Provide an injection interface to others and save it to local callback stack
+         */
+        that.onSortChange = function (callbackFn) {
```

Comment:
renaming it to something like `addOnSortChange` should be considered

Reply:
I'd either keep it as onSortChange or change to addSortChangeListener (similar to the DOM method addEventListener). I don't have a strong preference for either.

Reply:
I think "**addSortChangeListener**" as a name for this function might be a good choice so that it won't be confused with the "onSortChange" function from pfSort itself. Would it be fine? @matedo1

Reply:
+1 for `addSortChangeListener`